### ISSUE TITLE
Library issues with New MyDSL Project Wizard in 2.14

### ISSUE THREAD

**Issue by NDuemmel** [eclipse/xtext-core#851]
Hello,

creating a new MyDSL Project using the New Project Wizard that came with 2.14 leads to issues with resolving library contents offered by MyDSL. The contents cannot be resolved even after adding the dependency in the manifest.

My DSL Plugin has these line in mwe2 file:

```
projectWizard = {
    generate = true
}
fileWizard = {
    generate = true
}
```

When I create my Project via New -> MyDSL Project, it lacks the dependency to MyDSL Plugin after creation and its .project looks like this:

```
<?xml version="1.0" encoding="UTF-8"?>
<projectDescription>
	<name>qwert</name>
	<comment></comment>
	<projects>
	</projects>
	<buildSpec>
		<buildCommand>
			<name>org.eclipse.jdt.core.javabuilder</name>
			<arguments>
			</arguments>
		</buildCommand>
	</buildSpec>
	<natures>
		<nature>org.eclipse.jdt.core.javanature</nature>
		<nature>org.eclipse.pde.PluginNature</nature>
		<nature>org.eclipse.xtext.ui.shared.xtextNature</nature>
	</natures>
</projectDescription>
```

Adding these lines to .project manually, the dependency to manifest and restarting the runtime environment seems to do the trick. 

```
		<buildCommand>
			<name>org.eclipse.xtext.ui.shared.xtextBuilder</name>
			<arguments>
			</arguments>
		</buildCommand>
```

Best regards,
Nikita

**Comment by ArneDeutsch** [comment]
Fixed the default project template. In the meantime until the release you can adapt the generated `MyDslProjectTemplateProvider` and add the builder `org.eclipse.xtext.builder.impl.XtextBuilder.BUILDER_ID` there. You will have to adapt the template anyway.

**Comment by NDuemmel** [comment]
Thank you, it works now.

**Comment by cdietrich** [comment]
Fixed in 2.15
